verify/type_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import "testing"
+
+func Test_trimTitle(t *testing.T) {
+	tests := []struct {
+		name  string
+		title string
+		want  string
+	}{
+		{
+			name:  "regular PR title",
+			title: "📖 book: Use relative links in generate CRDs doc",
+			want:  "📖 book: Use relative links in generate CRDs doc",
+		},
+		{
+			name:  "PR title with WIP",
+			title: "WIP 📖 book: Use relative links in generate CRDs doc",
```

Comment:
Would we allow WIP to me merged?

Reply:
I think it's just not the responsibility of this check to also produce errors for that case (previous version of that code allowed it too).

Independent of that GitHub check the Prow WIP label plugin adds a `do-not-merge/work-in-progress` label. Tide is then configured to block on this label.

Example:
* https://github.com/kubernetes-sigs/cluster-api/pull/6000
* https://prow.k8s.io/pr?query=is%3Apr+repo%3Akubernetes-sigs%2Fcluster-api+author%3Aschrej+head%3Aproposal%2Fipam